src/modules/BuilderPlace/queries.ts

```diff
@@ -8,15 +9,26 @@ export const getBuilderPlace = async (domain: string) => {
   if (!builderPlace) {
     throw new Error(`Builder Place not found for domain ${domain}`);
   }
+  const userResponse = await getUserById(
+    Number(process.env.NEXT_PUBLIC_DEFAULT_CHAIN_ID),
+    builderPlace.ownerTalentLayerId,
+  );
+
+  if (!userResponse?.data?.data?.user) {
+    throw new Error(`Current user not found for id ${builderPlace.ownerTalentLayerId}`);
+  }
 
   const serializedBuilderPlace = JSON.parse(JSON.stringify(builderPlace));
+  const serializedConnectedUser = JSON.parse(JSON.stringify(userResponse.data.data.user));
 
   console.log({
     serializedBuilderPlace,
+    serializedConnectedUser,
   });
   return {
     props: {
       builderPlace: serializedBuilderPlace,
```

Comment:
as it's really closely link to the builderPlace, I think it's better to put builderPlace.user = ...

Reply:
up